src/commands/remove-demo.ts

```diff
@@ -0,0 +1,98 @@
+import { GluegunToolbox } from "gluegun"
+import { demo } from "../tools/demo"
+import type { CommentType } from "../tools/demo"
+import { createGetAllFilePaths } from "../tools/path"
+import { p, warning } from "../tools/pretty"
+
+module.exports = {
+  alias: ["rd", "remove-demos"],
+  description: "Remove demo code from generated boilerplate",
+  run: async (toolbox: GluegunToolbox) => {
+    const { parameters, patching, filesystem } = toolbox
+
+    const CWD = process.cwd()
+    const TARGET_DIR = parameters.first ?? CWD
+    p()
+    p(`Removing demo code from '${TARGET_DIR}'`)
+
+    const getAllFilePaths = createGetAllFilePaths(filesystem)
+    const paths = getAllFilePaths(TARGET_DIR)
+
+    // Go through every file path and handle the operation for each demo comment
+    const demoCommentResults = await Promise.allSettled(
+      paths.map(async (path) => {
+        const { exists, update } = patching
+
+        const comments: CommentType[] = []
+
+        if (await exists(path, demo.CommentType.REMOVE_FILE)) {
+          filesystem.remove(path)
+          comments.push(demo.CommentType.REMOVE_FILE)
+          return { path, comments }
+        }
+
+        if (await exists(path, demo.CommentType.REMOVE_CURRENT_LINE)) {
+          await update(path, demo.removeCurrentLine)
+          comments.push(demo.CommentType.REMOVE_CURRENT_LINE)
+        }
+
+        if (await exists(path, demo.CommentType.REMOVE_NEXT_LINE)) {
+          await update(path, demo.removeNextLine)
+          comments.push(demo.CommentType.REMOVE_NEXT_LINE)
+        }
+
+        if (
+          (await exists(path, demo.CommentType.REMOVE_BLOCK_START)) &&
+          (await exists(path, demo.CommentType.REMOVE_BLOCK_END))
+        ) {
+          await update(path, demo.removeBlock)
+          comments.push(demo.CommentType.REMOVE_BLOCK_START, demo.CommentType.REMOVE_BLOCK_END)
+        }
+
+        return { path, comments }
+      }),
+    )
+
+    // Handle the results of the demo comment operations
+    demoCommentResults
+      // Sort the results by the path in alphabetical order
+      .sort((a, b) => {
+        if (a.status === "fulfilled" && b.status === "fulfilled") {
+          return a.value.path.localeCompare(b.value.path)
+        }
+        return 0
+      })
```

Comment:
Since we don't have a guarantee that files will processed in the same order each time we run the comment, in order to have a consistent log output for our snapshot test, we are sorting the results by file path